gbpservice/nfp/configurator/lib/fw_constants.py

```diff
@@ -13,10 +13,15 @@
 DRIVERS_DIR = 'gbpservice.nfp.configurator.drivers.firewall'
 SERVICE_TYPE = 'firewall'
 VYOS = 'vyos'
+PALOALTO = 'paloalto'
```

Comment:
All vendor specific constants should move out of this file and go into a new constants file (say paloalto_fw_contants.py) which can be kept along with the driver file. Please check vyos files for example.